styles/general.css

```diff
@@ -6,11 +6,17 @@
     box-sizing: border-box;
     margin: 0;
     padding: 0;
+    color: white;
+}
+html, body {
+    height: 100%;
 }
 
 body {
     background:rgb(26, 34, 48);
     background: linear-gradient(0deg, rgba(26,34,48,1) 0%, rgba(20,19,16,1) 100%);
     background-attachment: fixed;
     font-family:Oswald, Arial;
```

Comment:
Thought we were using Inter as a font

Reply:
Yep the font is just a placeholder. We can always change it if needed